src/librustc/ty/layout.rs

```diff
@@ -888,6 +888,11 @@ impl<'a, 'tcx> LayoutCx<'tcx, TyCtxt<'a, 'tcx, 'tcx>> {
                     if x < min { min = x; }
                     if x > max { max = x; }
                 }
+                // We might have no inhabited variants, so pretend there's at least one.
+                if (min, max) == (i128::max_value(), i128::min_value()) {
```

Comment:
Just to make sure, is `(min, max) = max(), min()` correct?

Reply:
Jup. These are the initial values for both, so that means the loop was empty or all variants uninhabited